Cargo.toml

```diff
@@ -1,18 +1,5 @@
 [workspace]
-members = [
-    "crates/edr_defaults",
-    "crates/edr_eth",
-    "crates/edr_evm",
-    "crates/edr_napi",
-    "crates/edr_provider",
-    "crates/edr_rpc_client",
-    "crates/edr_rpc_eth",
-    "crates/edr_scenarios",
-    "crates/edr_solidity",
-    "crates/edr_test_utils",
-    "crates/edr_utils",
-    "crates/tools",
-]
+members = ["crates/*"]
```

Comment:
I recall @agostbiro purposefully made this per-crate as some of the folders inside `crates/*` (in the Solidity test runner branch) are not supposed to be included as crates.

Reply:
If I understand the issue correctly I think we could use:

```toml
members = ["crates/*", "crates/foundry/*"]
exclude = ["crates/foundry"]
```

Reply:
Unfortunately the issue with wildcards is that if you add a crate in a branch, and then switch branches, the directory may remain if it has ignored files and that breaks cargo.